bin/beeline

```diff
@@ -23,8 +23,29 @@
 # Enter posix mode for bash
 set -o posix
 
+realpath () {
```

Comment:
I think if we move this function to a new script, there still has a problem of how to find the real path of this new script.